full-stack-tests/backend/src/integration/AzuriteTest.ts

```diff
@@ -53,6 +54,58 @@ export namespace AzuriteTest {
       return blobClient.exists();
     };
 
+    export const subtractFromCurrentWriteLockExpiryTime = async (container: CloudSqlite.CloudContainer, blockName: string, numOfMin: number) => {
```

Comment:
can blockName just be hardcoded in this function? I think the fxn only makes sense if we're working with bcv_kv.bcv

Reply:
cloud just make it optional with default, get the best of both worlds

Reply:
Resolved in [commit b6de82106a3f57bb9db95b4dbd3dcfd47076e2c8](https://github.com/iTwin/itwinjs-core/pull/7000/commits/b6de82106a3f57bb9db95b4dbd3dcfd47076e2c8)